### DISCUSSION TITLE
Solid to "floor" but not to other sprites

### DISCUSSION THREAD

**Discussion by nini1294** [replit/kaboom#586] [Q&A]
Hi, I'm net to kaboom and wanted to know if there is a way to make a sprite solid = true for walls and floors, but not to other sprites with a certain tag.

**Comment by lajbel** [comment]
Hey hi!

Sorry for the delay, currently I don't think there is anything like collision tags, maybe you can use different layers, since between different layers collisions are not shared.

https://kaboomjs.com/play?demo=layer
